cmd/kube-rbac-proxy/app/kube-rbac-proxy.go

```diff
@@ -303,25 +287,26 @@ func secureServerRunner(
 	return runner, interrupter
 }
 
-func setupAuthorizer(krbInfo *server.KubeRBACProxyInfo, delegatedAuthz *serverconfig.AuthorizationInfo) (authorizer.Authorizer, error) {
-	staticAuthorizer, err := static.NewStaticAuthorizer(krbInfo.Authorization.Static)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create static authorizer: %w", err)
+func setupAuthorizer(krpInfo *server.KubeRBACProxyInfo, delegatedAuthz *serverconfig.AuthorizationInfo) (authorizer.Authorizer, error) {
```

Comment:
Not sure if applicable, but I renamed `krbInfo` to `krpInfo` to use the initials.

Reply:
Thanks for the clean-up. I assume it is a typo (someone had "kube RBac proxy" in mind?).